[Docs-2.0] FAQ-UseWasabi small update 1 (#1095)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

* As a 13th word (passphrase) as described in [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).
* To encrypt the private key of the extended private key as described in [BIP 38](https://github.com/bitcoin/bips/blob/master/bip-0038.mediawiki) to get an encrypted secret which is stored on the computer.

> Wasabi stores only the BIP38 encrypted blob, so you'll need to type in the password to spend or coinjoin from Wasabi.

Reviewer: ```suggestion
Wasabi stores only the BIP38 encrypted blob, so you'll need to type in the password to open the wallet and to spend from Wasabi.
```

:::details
### How do I know if the synchronization is finished?

> You know that Tor and backend is properly connected when you see a checkmark at the bottom right corner of the synchronization page.

Reviewer: ```suggestion
You know that Tor and the backend are properly connected when you see a check mark at the bottom right corner of the synchronization page.
```

> You know that Tor and the backend are properly connected when you see a check mark at the bottom right corner of the synchronization page.

Reviewer 2: ```suggestion
Once Tor and backend are properly connected and you have peers, you will see a checkmark at the bottom right corner of the main view.
```

> All the block filters and all the relevant blocks are downloaded and the wallet is synchronized when the timer of the page runs out.

Reviewer: ```suggestion
The timer of the page runs out when all the block filters and all the relevant blocks are downloaded and the wallet is synchronized.
```
